### ISSUE TITLE
Graphs too big for really strong capacitors

### ISSUE THREAD

**Issue by Xalcon** [Xalcon/AnalyzeIO#5]
Graphs will render over the boundaries of the graph view when reaching level 4 or above.
While writing the mod, I was able to find capacitors that had modifiers that went as high as level 10.

We need to find a better way to render these.
* dynamic scaling: all graphs get scaled down equal to the maximum modifier level on a given capacitor.
* Rework graphs to render on the full width of the gui and expect capacitors up to a certain level (level 6?) and limit graphs to not render bigger than that. Modifiers that have an effective level of 6+ would then get a different color or some form of identifier (icon, actual number, etc)

**Comment by Xalcon** [comment]
It looks like reddit found a way to create capacitors that will break any gui (increasing the amount of RF/t generated from a single piece of coal to 500 million per tick) This cant be displayed with a gui, but since this can be considered a bug, I will wait for the developers to respond to this bug.
The GUI update will not happen before EnderIO released a stable version for 1.11.2 (or higher) since I do not intend on doing much more work on any MC 1.10.2 versions other than bug fixes.

**Comment by HenryLoenwind** [comment]
BTW, max valid level is 4.75 for loot caps.

Most scalers are limited to 5, a few technically unlimited (e.g. area). So third-party caps could have higher values---but you're only allowing Ender IO caps into your machine anyway, I've seen.

**Comment by Xalcon** [comment]
Do you know of any mods that add custom capacitors? I was not aware there were any.

**Comment by HenryLoenwind** [comment]
No, I have not yet seen some. Which is a shame, I added the nbt-based system because it was requested so often... (The loot caps were only intended as an example of how to use it.)
